src/modules/Builder/features/ActivityItems/ItemConfiguration/GroupedSelectSearchController/ItemTypeTooltip/TooltipComponents/TimeRange.tsx

```diff
@@ -18,7 +18,7 @@ export const TimeRange = () => {
 
   return (
     <>
-      <StyledPresentation>
+      <StyledPresentation data-testid="tooltip-selection-presentation">
```

Comment:
please add a variable for `data-testid="tooltip-selection-presentation"` in constants in parent folder of these component